src/Player/Player.tsx

```diff
@@ -58,11 +58,8 @@ const Player: React.FC<Props> = ({ currentRecord }) => {
         mb: 3,
       }}
     >
-      <Box
-        sx={{
-          height: '300px',
```

Comment:
This was making the progress bar hide behind the video for some reason so I removed it, but now the video is small.